hrms/payroll/doctype/employee_tax_exemption_proof_submission_detail/employee_tax_exemption_proof_submission_detail.json

```diff
@@ -51,7 +52,13 @@
    "in_list_view": 1,
    "label": "Actual Amount",
    "options": "currency"
-  }
+  },
+  {
+    "fieldname": "attach_proof",
+    "fieldtype": "Attach",
+    "label": "Attach Proof"
```

Comment:
enable `in_list_view` for the field.
Also, you will have to update modified timestamp for the doctype for its changes to sync to sites on migrate. Always try making doctype changes from the UI so all relevant properties get updated

Reply:
I think we should also hide the old attachment field (in the parent) in the js controller if it has no value. This will ensure:
- Users attach proof against individual rows in new proof submissions
- For old proof submissions, if attachments are already present, the field will be shown.